docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FieldAwareFactorizationMachinewWithOptions.cs

```diff
@@ -1,84 +1,167 @@
 ﻿using System;
+using System.Collections.Generic;
```

Comment:
This file is produced by TT. Please ignore the difference here because it's caused by a renaming.